admin_guide/out_of_resource_handling.adoc

```diff
@@ -75,14 +170,23 @@ curl <certificate details> \
 |`imagefs.inodesFree` = `node.stats.runtime.imagefs.inodesFree`
 |===
 
-The node supports two file system partitions when detecting disk pressure.
+Each of the above signals supports either a literal or percentage-based value. The percentage-based value is calculated relative to the total capacity associated with each signal.
+
+A script derives the value for `memory.available` from the `cgroupfs` driver using the same set of steps that the kubelet performs. The script excludes *inactive_file* memory (that is, the number of bytes of file-backed memory on inactive LRU list) from its calculation as it assumes that *inactive_file* memory is reclaimable under pressure. 
```

Comment:
@qwang1 Changed this to: "from your `cgroup` driver." Does this make more sense?